addon/components/swiper-container.js

```diff
@@ -101,6 +101,11 @@ export default Component.extend({
       options.autoplayDisableOnInteraction = this.get('autoplayDisableOnInteraction');
     }
 
+    if (this.get('watchSlidesVisibility')) {
+      options.watchSlidesProgress = true;
```

Comment:
I can add the `whatSlideProgress` option also however I think a user should be able to pass one option to turn on `watchSlidesVisibility` functionality